apps/braze/src/components/create/CreateFlow.tsx

```diff
@@ -20,12 +20,23 @@ type CreateFlowProps = {
 
 const CreateFlow = (props: CreateFlowProps) => {
   const { sdk, entry, initialSelectedFields = [] } = props;
-
   const [step, setStep] = useState(FIELDS_STEP);
   const [selectedFields, setSelectedFields] = useState<Set<string>>(new Set(initialSelectedFields));
   const [contentBlockName, setContentBlockName] = useState('');
   const [isSubmitting, setIsSubmitting] = useState(false);
 
+  const allowedFields = entry.fields.filter((field) => field.isEnabledForCreate());
+
+  const filteredEntry = new Entry(
```

Comment:
If everything is the same except for the fields, we could just set the fields in the original entry
```suggestion
  entry.fields = entry.fields.filter((field) => field.isEnabledForCreate());
```

Reply:
I wasn't sure if we wanted the entry to be filtered in others steps, so I didn't want to change the original entry. But it looks like we don't need the rest of the fields then. ✅